### DISCUSSION TITLE
How to remove cat emoji from the title bar?

### DISCUSSION THREAD

**Discussion by wishnish** [kovidgoyal/kitty#6990] [Q&A]
![IMG20240112163203.jpg](https://github.com/kovidgoyal/kitty/assets/103920294/35a8cd36-7362-4a4c-ba83-c55bfbc70fd4)

How can I remove that cat emoji?
Top left corner
Please anyone guide me

I'm using Fedora 39

If I do hide decorations in kitty.conf i cannot resize the terminal window 

https://github.com/kovidgoyal/kitty/assets/103920294/53f4d0ab-d870-41a1-959e-1ae39a08557e

@kovidgoyal  🙏

**Comment by cybardev** [comment]
Have you tried setting a `tab_title_template`?

If not, try that. Here's the documentation: https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.tab_title_template

**Reply by sk-panigrahi** [reply]
i checked it out. it did not work for me.. it is for changing how the tab titles are displayed.
